plugins/catalog-react/src/components/EntityNamespacePicker/EntityNamespacePicker.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2021 The Backstage Authors
```

Comment:
```suggestion
 * Copyright 2023 The Backstage Authors
```
There's a few new files in the PR that have the wrong year — apologies for the nit!